### PR TITLE
wmco: Add CI jobs for community 4.13

### PR DESCRIPTION
**Pull request by alinaryan**
wmco: Add configs and conditionally skip CI jobs for community-4.13
core-services: onboard community-4.13 branch

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.13__okd.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+base_images:
+  openshift_release_golang-1.20:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
```
[the remark this answers is not in the document]
Comment (saifshaikh48) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.13__okd.yaml:5]: https://issues.redhat.com/browse/ART-5546

**Review by aravindhp — APPROVED** [review]

**Pull request merged**